controller/controller_test.go

```diff
@@ -353,7 +353,7 @@ func TestController_Submit_Error(t *testing.T) {
 	// Assert
 	Expect(task).To(BeNil())
 	Expect(err).To(HaveOccurred())
-	Expect(err.Error()).To(Equal("invalid argument: units must not be empty"))
+	Expect(err.Error()).To(Equal("no units in group"))
```

Comment:
is it possible to compare this to `noUnitsInGroupError ` rather than the string. in case of changing the error message we would just change it in one place.